micro_sam/sam_annotator/__init__.py

```diff
@@ -2,8 +2,8 @@
 The interactive annotation tools.
 """
 
-from .annotator import annotator
-from .annotator_2d import annotator_2d
-from .annotator_3d import annotator_3d
-from .annotator_tracking import annotator_tracking
-from .image_series_annotator import image_folder_annotator, image_series_annotator
+# from .annotator import annotator
+# from .annotator_2d import annotator_2d
```

Comment:
Why are these lines commented out?

This is what caused`test_gui.py` to fail, since it used to do `from micro_sam.sam_annotator import annotator_2d`. I've changed the import in `test_gui.py` to now say `from micro_sam.sam_annotator.annotator_2d import annotator_2d`. So now it should pass, regardless of whatever you decide to do here.

Reply:
I had to comment these out locally in order to fix the vigra import issues.

Reply:
Feel free to undo these changes. I assume these import errors are specific to my installation and I will look into this more closely at some later point (and first see if this still persists with a new environment.)